app/src/main/java/com/example/wnews/models/Comment.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.wnews.models
+
+data class Comment(
+    var id : Int = 0,
+    var commenter : String = "",
```

Comment:
I would suggest to name this variable `commenterId`, so it makes it more clear that the String represents an ID.